### PR TITLE
Auth page

### PR DESCRIPTION
**Pull request by rahulyadav139**
Demo: https://statuesque-creponne-891db8.netlify.app/

### CODE REVIEW

File: src/components/authorization/AuthForm.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  flex-direction: row;
+}
+
+.main section {
+  padding: 2rem;
+  flex-basis: 50%;
+}
+.main img {
+  object-fit: contain;
+} */
```
Comment (dipen2000) [inline src/components/authorization/AuthForm.css:12]: remove the commented code

File: src/components/authorization/AuthForm.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+/* .form .wrapper-username div {
+  flex-grow: 1;
+} */
+
+/* .form .form__greet-msg {
+  font-size: 1.2rem;
+  color: var(--primary-color-dark);
+  font-weight: bold;
```
Comment (dipen2000) [inline src/components/authorization/AuthForm.css:44]: here too.

**Review by dipen2000 — APPROVED** [review]
Nothing much to add here the code is structured and clean. good to go.

**Review by j24m — APPROVED** [review]
Hello, I like the code, it's understandable and simple.
I liked the UI as well. Good work.

**Comment by rahulyadav139** [comment]
> Hello, I like the code, it's understandable and simple. I liked the UI as well. Good work.

Thank you.

File: src/components/authorization/AuthForm.css (hunk 2 above)
Comment (rahulyadav139) [reply src/components/authorization/AuthForm.css:44]: My bad. I forgot to do that. I'll remove it.

**Pull request merged**